auslib/admin/views/forms.py

```diff
@@ -168,7 +169,7 @@ class PartialReleaseForm(Form):
 
 
 class RuleForm(Form):
-    backgroundRate = IntegerField('Background Rate', validators=[Required(), NumberRange(0, 100)])
+    backgroundRate = IntegerField('Background Rate', validators=[any_of(range(0, 101, 1))])
```

Comment:
Is there a reason you're using any_of() instead of NumberRange()? As I mentioned before, NumberRange seems to do the same thing, except it provides a better error message.